tests/test_vec_normalize.py

```diff
@@ -202,6 +202,33 @@ def test_runningmeanstd():
         assert np.allclose(moments_1, moments_2)
 
 
+def test_combining_stats():
+    np.random.seed(4)
+    for shape in [(1,), (3,), (3, 4)]:
+        values = []
+        rms_1 = RunningMeanStd(shape=shape)
+        rms_2 = RunningMeanStd(shape=shape)
+        rms_3 = RunningMeanStd(shape=shape)
+        for _ in range(15):
+            value = np.random.randn(*shape)
+            rms_1.update(value)
+            rms_3.update(value)
+            values.append(value)
+        for _ in range(19):
+            rms_2.update(value)
```

Comment:
More nitpick, but I assume here we just want to skew the stats towards `value` for a test (away from normal distribution)?

Reply:
ah no, just me copy pasting code without thinking too much :see_no_evil: